woocommerce/wc-admin
# Add CES survey for adding product categories, tags, and attributes

This PR adds CES survey trigger on adding product categories and tags.

Partially fixes https://github.com/woocommerce/woocommerce-admin/issues/6307.

### Detailed test instructions:

- Make sure tracking is enabled in settings: 
```
/wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com
```
- Delete the option `woocommerce_ces_shown_for_actions` to make sure CES prompt triggers when updating settings.
- Enable the logging of Tracks events to your browser dev console:
```
localStorage.setItem( 'debug', 'wc-admin:tracks' );
```

**Testing product categories:**
- Go to Products > Categories.
- Add a new category.
- Observe CES prompt "How easy was it to add a product category?" is displayed.

**Testing product tags:**
- Go to Products > Tags.
- Add a new tag.
- Observe CES prompt "How easy was it to add a product tag?" is displayed.

**Testing product attributes:**
- Go to Products > Attributes.
- Add a new attribute.
- Observe CES prompt "How easy was it to add a product attribute?" is displayed.
